CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.0.0 (02/12/2019)
+
+**Common**
+
+* Project structure was changed, public API aligned to the HERE OLP SDK for C++.
+* Unit tests was improved
```

Comment:
Not sure we should write this. I would not write this.

Reply:
Removed